pcapng/structs.py

```diff
@@ -3,20 +3,31 @@
 """
 
 import abc
-import io
 import struct
 import warnings
 
-import six
-
-from pcapng.exceptions import BadMagic, CorruptedFile, StreamEmpty, TruncatedFile
-from pcapng.utils import unpack_euiaddr, unpack_ipv4, unpack_ipv6, unpack_macaddr
-
-try:
-    from collections.abc import Mapping
-except ImportError:
-    from collections import Mapping
-
+from pcapng import strictness as strictness
+from pcapng.exceptions import (
+    BadMagic,
+    CorruptedFile,
+    PcapngLoadError,
+    StreamEmpty,
+    TruncatedFile,
+)
+from pcapng.flags import FlagBool, FlagEnum, FlagField, FlagUInt, FlagWord
+from pcapng.utils import (
+    Iterable,
+    Mapping,
```

Comment:
Avoid wrapping these around, import them from `collections` (the deprecation shouldn't be issuing warnings right now.)